handlers/user.go

```diff
@@ -0,0 +1,48 @@
+package handlers
```

Comment:
UsersHandler and UserHandler should not be in separate files. Rather, there should be a User controller that is responsible for all methods relating to User objects.